modules/vardictjava/main.nf

```diff
@@ -0,0 +1,43 @@
+process VARDICTJAVA {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::vardict-java=1.8.3" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/vardict-java:1.8.3--hdfd78af_0':
+        'quay.io/biocontainers/vardict-java:1.8.3--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(bam), path(regions_of_interest)
+    path(reference_fasta)
+
+    output:
+    tuple val(meta), path("*.vcf"), emit: vcf
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    """
+    head -n 20 $reference_fasta
```

Comment:
This is probably an artefact from testing?

Reply:
Yes, forgot to delete it